doc/lsst.daf.butler/datastores.rst

```diff
@@ -28,7 +28,8 @@ The supported datastores are:
 File-Based Datastores
 =====================
 
-The file-based datastores (for example `~datastores.posixDatastore.PosixDatastore` and `~datastores.s3Datastore.S3Datastore`) share configuration since they use formatters to serialize datasets to file artifacts using file name template schemes and also support disassembly of :ref:`composite storage classes <daf_butler_storage_classes>`.
+There is a single file-based datastore (`~datastores.fileDatastore.FileDatastore`) that handles local POSIX file system and remote object stores.
```

Comment:
The main criteria for this datastore are that every URI corresponds to a single (possibly composite) dataset and, vice versa, that every dataset is labeled by a URI?  (As opposed to, say, a SQL-database-like datastore where multiple queries, even if expressed as URIs, might correspond to the same effective dataset.)

Reply:
I've added an explicit comment about that.